packages/opentelemetry-cloud-monitoring-exporter/src/external-types.ts

```diff
@@ -47,6 +47,12 @@ export interface ExporterOptions {
    * monitoring.googleapis.com:443.
    */
   apiEndpoint?: string;
+  /**
+   * Assume all metric descriptors have already been created and publish
+   * metrics without checking. This can prevent hitting a rate limit in Google
+   * when a large number of clients are all started up at the same time.
+   */
+  skipDescriptorCheck?: boolean;
```

Comment:
On second thought, the option in Go is called [`DisableCreateMetricDescritpors`](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/82615d06c71484903ae6bfe917f87d66e76c00fe/exporter/metric/option.go#L142C6-L142C40). Can you rename this to `disableCreateMetricDescriptors` instead?

Please also update the PR name as well to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) and use the same phrasing, since it ends up in the changelog.

Reply:
Done